charge.go

```diff
@@ -689,15 +672,6 @@ type ChargeOutcome struct {
 	Type          string             `json:"type"`
 }
 
-// ShippingDetails is the structure containing shipping information.
-type ShippingDetails struct {
-	Address        *Address `json:"address"`
-	Carrier        string   `json:"carrier"`
-	Name           string   `json:"name"`
-	Phone          string   `json:"phone"`
-	TrackingNumber string   `json:"tracking_number"`
-}
-
```

Comment:
Whis is this disappearing?

Reply:
Re-added - it should've been manually specified in address.go.